assets/schemas/manifest.json

```diff
@@ -1566,6 +1642,167 @@
         }
       }
     },
+    "tcf": {
+      "type": "object",
+      "vendor":{
```

Comment:
It doesn't work this way. You need to put the properties inside `"properties": {}`